invenio_migrator/legacy/deposit.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2016 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Invenio-Deposit migration."""
+
+from __future__ import absolute_import, print_function
+from itertools import islice
+import json
+from invenio.modules.deposit.models import Deposition
+
+
+def default_serializer(o):
+    """Default serializer for json."""
+    import datetime
+    defs = (
+        ((datetime.datetime, datetime.date, datetime.time),
+         lambda x: x.isoformat(), ),
```

Comment:
``from .utils import datetime_toutc``

This ensures we get timestamps with timezone information.